### DISCUSSION TITLE
Week 1 Xion_Nights

### DISCUSSION THREAD

**Discussion by chocolatmint** [teamalgoritma/community#810] [Q&A] [answered]
Pada sesi dive deeper kemarin malam, disebutkan bahwa dataframe spotify memiliki Release Date. 
<img width="384" alt="image" src="https://github.com/teamalgoritma/community/assets/9330628/1986eed2-96d9-4f7c-8de1-6692daecf090">
Pertanyaan saya:
1. Apakah sudah betul penggunaan tipe data object di Release Date, atau sebaiknya diubah menjadi datetime64?
2. Apakah ada syntax pandas yang equivalent dengan query sql seperti berikut?
where date <= "yyyy-MM-dd" and date > "yyyy-MM-dd"?
Jika ada syntax yang equivalent dengan sql tersebut, tipe data pandas apakah yang harus saya gunakan di kolom date?

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @chocolatmint ,

Terima kasih telah membuat pertanyaan sebelum datang di sesi mentoring. Berdasarkan hal yang telah di bawah di sesi mentoring, untuk solve permasalahan ini antara lain dengan:

- Mengubah kolom menjadi tipe data datetime dengan `.astype("datetime64[ns]")`
- Setelah itu kita baru bisa melakukan conditional subsetting berdasarkan rentang waktunya.

Berhubung permasalahan ini telah diselesaikan saat sesi mentoring, maka dari itu kami akan menandai jawaban ini sebagai "Mark as Answer". Terima kasih.

Best Regards,
Algoritma
